Dockerfile

```diff
@@ -1,12 +1,26 @@
-FROM ubuntu:trusty
+FROM quay.io/sameersbn/ubuntu:latest
+MAINTAINER Alexander Trost <galexrt@googlemail.com>
 
-EXPOSE 9991
+ENV ZULIP_VERSION="1.3.10" DATA_DIR="/data"
 
-RUN apt-get update && apt-get install -y \
-  python-pbs \
-  wget
+RUN apt-get -q update && \
+    apt-get -q dist-upgrade -y && \
+    mkdir -p "$DATA_DIR" /root/zulip && \
+    wget -q "https://www.zulip.com/dist/releases/zulip-server-$ZULIP_VERSION.tar.gz" -O /tmp/zulip-server.tar.gz && \
+    tar xfz /tmp/zulip-server.tar.gz -C /root/zulip --strip-components=1 && \
+    rm -rf /tmp/zulip-server.tar.gz && \
+    export VOYAGER_CLASS="dockervoyager" DEPLOYMENT_TYPE="dockervoyager" ADDITIONAL_PACKAGES="python-dev python-six python-pbs" && \
+    /root/zulip/scripts/setup/install && \
+    chown zulip:zulip /opt/createZulipAdmin.sh && \
+    apt-get -qq autoremove --purge -y && \
+    apt-get -qq clean && \
+    rm -rf /root/zulip/puppet/ /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-RUN useradd -d /home/zulip -m zulip && echo 'zulip ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+COPY docker-entrypoint.sh /sbin/entrypoint.sh
+COPY setup_files/ /opt/files
 
-USER zulip
-WORKDIR /srv/zulip
+VOLUME ["$DATA_DIR"]
+EXPOSE 80 443
+
+ENTRYPOINT ["/sbin/entrypoint.sh"]
+CMD ["app:run"]
```

Comment:
Renaming so the production docker file has the "Dockerfile" name is potentially fine, but it might be better to have "Dockerfile-prod" and "Dockerfile.dev" for clarity.  Regardless, I think the README.dev.md instructions for using the old development Dockerfile need to be updated to specify the path to the Dockerfile....